world/board.go

```diff
@@ -87,7 +87,7 @@ func (b *board) unitEat(u unit) []interface{} {
 	var response []interface{}
 	eatenItem := t.occupants[unitTypeItem]
 	if eatenItem != nil {
-		fmt.Println("ate", eatenItem._name(), "at", b.units[u][0], b.units[u][1])
+		fmt.Println("ate", eatenItem._name() , "at", b.units[u][0], b.units[u][1])
```

Comment:
Did I forget to gofmt or did you lol
Same with main.go:20

Reply:
probably you, because i didnt actually edit that code LMFAO, just removed lines of merge conflict